### ISSUE TITLE
All MS Basic documentation still references MuseScore_General

### ISSUE THREAD

**Issue by Declan-Griner** [musescore/MuseScore#19446]
### Issue type

Choose option...

### Bug description

All documentation for the MS Basic soundfont still references the previous MuseScore_General. This error occurs in MS Basic_Readme.md, MS Basic_License.md, the README.md file in the sounds folder, and in the comments in the .sf3 file itself when viewed in Polyphone.

### Steps to reproduce

The issue is apparent when viewing one of the aforementioned files

### Screenshots/Screen recordings

![Sf3_file_error](https://github.com/musescore/MuseScore/assets/143758845/14b36fd8-f032-4ac1-98d0-56335def5ca0)
This appears after opening MS Basic.sf3 in Polyphone

![README_Error](https://github.com/musescore/MuseScore/assets/143758845/6bee9a22-9f61-443c-9c01-5e56797ee2c5)
Appears in README.md in the 'sounds' folder

![MS_Basic_Licesnse_Error](https://github.com/musescore/MuseScore/assets/143758845/df95f7ca-89a5-4511-8c7e-16af37379484)
Appears in the MS Basic_License.md file

![MS_Basic_Changelog_Error](https://github.com/musescore/MuseScore/assets/143758845/f1f7f1ff-c051-4461-bf84-44d775298308)
Appears in  MS_Basic_Changelog.md

### MuseScore Version

N/A

### Regression

I don't know

### Operating system

Fedora 38

### Additional context

Appears that this is a holdover from the previous name for the .sf3 file

**Comment by vashutangri** [comment]
Hey, I would love to work on this issue

**Comment by vashutangri** [comment]
So should I change it to "Musescore"?

**Comment by cbjeukendrup** [comment]
No, to `MS Basic`, because that is the current name of the SoundFont (it was renamed from MuseScore_General to MS Basic).

**Comment by LeulProgramming** [comment]
If this is still an issue, I am perfectly fine with working on it.

**Comment by cbjeukendrup** [comment]
I removed the "community" tag from this issue; it looks like the documentation of the sound font needs some more rigorous updates, which should probably be done by someone from the in-house team who knows the history of it.
